### PR TITLE
fix(list view): Make the plus symbol clickable ( #1246).

### PR DESCRIPTION
**Pull request by nimishamukherjee**
Make the plus symbol click-able and display the work item types page. the The plus symbol appears in the work item list view when there are no work items.

### CODE REVIEW

**Comment by mindreeper2420** [comment]
Hi @nimishamukherjee 

In order to keep this PR in-line with the [PatternFly component](http://www.patternfly.org/pattern-library/communication/empty-state/), I would suggest removing the clickability from the '+' and adding a primary button (per the component designs) and have that perform the action to open the Work Item type page for adding a new Work Item. We can revisit this down the line (per our continuous iteration model) to determine what to do with other areas with empty states, in order to maintain consistency. UX will continue to evaluate how this is utilized as we move forward.

**Comment by mindreeper2420** [comment]
Also of note, the gray background should not be present - that is being pulled in from the Spaces template.

**Comment by nimishamukherjee** [comment]
@mindreeper2420 to make sure I understand what you are saying:
Based on http://www.patternfly.org/pattern-library/communication/empty-state/#/overview
I add the "Main Action" button, maybe with title "Add", clicking on this will initiate the detailed add flow.

**Comment by mindreeper2420** [comment]
@nimishamukherjee yes - I would make the text in the button read 'Create Work Item', so that it matches the Quick Add text.

**Comment by Mgranfie** [comment]
@mindreeper will this put focus on the quick add or the full details pane
will open?

**Comment by michaelkleinhenz** [comment]
@Mgranfie @mindreeper2420 can you mark the PR as "changes requested" if there need things to be worked on. It is hard to see from discussions if there is actually objections present. thx!

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 1]
```diff
@@ -52,7 +53,7 @@
                 <div class="col-sm-offset-2 col-md-offset-3 col-sm-8 col-md-6">
                   <div class="card-pf card-pf-aggregate-status">
                     <div class="card-pf-title">
-                      <h2 class="pficon pficon-add-circle-o no-wi"></h2>
+                      <h2 class="pficon pficon-add-circle-o no-wi with-cursor-pointer" (click)="typeSelectPanel.openPanel()"></h2>
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-list/work-item-list.component.html]: this should not be clickable. per the PatternFly Empty State component, there should be a clickable 'Create Work Item' primary button below all of the text.

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 2]
```diff
@@ -52,7 +53,7 @@
                 <div class="col-sm-offset-2 col-md-offset-3 col-sm-8 col-md-6">
                   <div class="card-pf card-pf-aggregate-status">
```
Comment (mindreeper2420) [inline src/app/work-item/work-item-list/work-item-list.component.html]: please remove the gray background that is present - according to the inspector, it is coming from the Spaces component scss file.

**Comment by nimishamukherjee** [comment]
@Preeticp, @mindreeper2420 once we add the blue Create Work Item button, we would have three options to create a work item.

createwi

The second line - "If you'd like to add a Work Item, you can do so using the Quick Add or the + in the bar above."
needs to be modified.

Concerns:
- The user does not understand what a quick add is
- The + icon appears in multiple places

**Comment by Preeticp** [comment]
A few suggestions/questions:

1. Do we need the black plus icon right in the middle of the page?
2. Maybe, the blue 'Create work item' button could replace the text in black, below the plus icon, 'Create a Work Item', making it obvious.
3. We could simply add 1 line saying, 'you could also use the Quick add (or the icon representing it) to create a Work Item. 
4. The Quick Add should have a unique icon which could be consistently used to represent smart/quick add actions throughout the UI.

.

**Comment by nimishamukherjee** [comment]
@michaelkleinhenz @mindreeper2420 can I close this PR. The original issue was about making the plus icon click-able and this involves a discussion with the PatternFly team.
Adding the Create Work Item button may complicate this issue further as the instruction text needs to be updated.
Your thoughts?

**Comment by mindreeper2420** [comment]
@nimishamukherjee @michaelkleinhenz I would close this and leave it as is, for the time being. All of the empty states need to be adjusted throughout the product.

**Comment by nimishamukherjee** [comment]
Closing this PR.

**Pull request closed without merging**